token_examples/factory_token/tests/common.rs

```diff
@@ -0,0 +1,245 @@
+    let bundle_root = bundle::import_bundle(&blockstore, actors_v9::BUNDLE_CAR).unwrap();
+
+    Tester::new(NetworkVersion::V15, StateTreeVersion::V4, bundle_root, blockstore.clone()).unwrap()
+}
+
+/// Helper routines to simplify common operations with a Tester
+pub trait TestHelpers {
+    /// Call a method on an actor
+    fn call_method(
+        &mut self,
+        from: Address,
+        to: Address,
+        method_num: u64,
+        params: Option<RawBytes>,
+    ) -> ApplyRet;
+
+    /// Call a method on an actor and assert a successful result
+    fn call_method_ok(
+        &mut self,
+        from: Address,
+        to: Address,
+        method_num: u64,
+        params: Option<RawBytes>,
+    ) -> ApplyRet;
+
+    /// Install an actor with initial state and ID
+    /// Returns the actor's address
+    fn install_actor_with_state<S: Serialize>(
+        &mut self,
+        path: &str,
+        actor_id: u64,
+        state: S,
+    ) -> Address;
+
+    /// Install an actor with no initial state
+    /// Takes ID and returns the new actor's address
+    fn install_actor_stateless(&mut self, path: &str, actor_id: u64) -> Address;
+}
+
+/// Helper routines to simplify common token operations
+pub trait TokenHelpers {
+    /// Get balance from token actor for a given address
+    /// This is a very common thing to check during tests
+    fn token_balance(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+    ) -> TokenAmount;
+
+    /// Mint tokens from token_actor to target address
+    fn mint_tokens(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+        amount: TokenAmount,
+        operator_data: RawBytes,
+    ) -> ApplyRet;
+
+    /// Mint tokens from token_actor to target address and assert a successful result
+    fn mint_tokens_ok(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+        amount: TokenAmount,
+        operator_data: RawBytes,
+    ) -> ApplyRet;
+
+    /// Check token balance, asserting that balance matches the provided amount
+    fn assert_token_balance(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+        amount: TokenAmount,
+    );
+    /// Check token balance, asserting a zero balance
+    fn assert_token_balance_zero(
+        &mut self,
+        operator: Address,
+        token_actor: Address,
+        target: Address,
+    );
+}
+
+impl<B: Blockstore, E: Externs> TestHelpers for Tester<B, E> {
+    fn call_method(
+        &mut self,
+        from: Address,
+        to: Address,
+        method_num: u64,
+        params: Option<RawBytes>,
+    ) -> ApplyRet {
+        static mut SEQUENCE: u64 = 0u64;
+        let message = Message {
+            from,
+            to,
+            gas_limit: 99999999,
```

Comment:
This is actually not that high. Can you import BLOCK_GAS_LIMIT to here from fvm_shared?

Reply:
Did this for the main integration tests too